OpenRA.Mods.Common/Traits/AmmoPool.cs

```diff
@@ -74,6 +79,8 @@ public AmmoPool(Actor self, AmmoPoolInfo info)
 
 			RemainingTicks = Info.SelfReloadDelay;
 			PreviousAmmo = GetAmmoCount();
+
+			pipText = GetAmmoCount() + "/" + Info.Ammo;
```

Comment:
Please set these only if the feature is enabled.  This otherwise generates unnecessary memory churn.

Reply:
Fixed and rebased. I wasn't able to check code style though...